portals/devportal/src/main/webapp/source/src/app/components/Apis/Chat/AISearchAssistant.jsx

```diff
@@ -69,18 +69,22 @@ function AISearchAssistant() {
                     let content;
                     try {
                         switch (error.response.status) {
-                            case 401: // Unauthorized
-                                content = 'Provided token is invalid. Please use a valid token and try again.';
+                            case 401:
+                                content = 'Apologies for the inconvenience. It appears that your token is invalid or expired. Please'
+                                + ' provide a valid token or upgrade your subscription plan.';
                                 break;
                             case 429: // Token limit exceeded
-                                content = 'Token limit is exceeded. Please try again later.';
+                                content = 'Apologies for the inconvenience. It appears that the token limit has been exceeded.'
+                                + ' Please attempt your request again.';
```

Comment:
Why do we ask the user to attempt the request again in a token limit hitting scenario?

Reply:
Merging this since we need a fixes soon, please add this fix for a subsequent PR